spec/lib/base/glooko_logging_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+RSpec.describe 'GlookoLogging' do
+  let(:logger_class) { Class.new { include Base::GlookoLogging } }
+
+  it 'logs expected output with json data' do
+    test = expect do
+      logger_class.new.log('warn', {fog: 'test'}, __LINE__, __method__,
```

Comment:
can we also test that the Logger received "warn" when logging the data?

Reply:
Please check if the latest change includes the requested test